dashx/src/main/java/com/dashx/sdk/DashXClient.kt

```diff
@@ -355,6 +356,76 @@ class DashXClient {
             })
     }
 
+    fun fetchStoredPreferences(
+        onSuccess: (result: JsonObject) -> Unit,
+        onError: (error: String) -> Unit
+    ) {
+
+        val fetchStoredPreferencesInput = FetchStoredPreferencesInput(
+            Input.fromNullable(this.uid),
+        )
+        val fetchStoredPreferencesQuery = FetchStoredPreferencesQuery(fetchStoredPreferencesInput)
+
+        apolloClient
+            .query(fetchStoredPreferencesQuery)
+            .enqueue(object : ApolloCall.Callback<FetchStoredPreferencesQuery.Data>() {
+                override fun onResponse(response: Response<FetchStoredPreferencesQuery.Data>) {
+                    val fetchStoredPreferencesResponse = response.data?.fetchStoredPreferences
```

Comment:
It should return the `preferencesData` within `response.data?.fetchStoredPreferences`